src/middleware/builtin.spec.ts

```diff
@@ -162,6 +162,30 @@ describe('matchMessage()', () => {
     );
     it('should filter out message events which do not have text (block kit)', noTextMessageTestCase(pattern));
   });
+
+  describe('using an array pattern', () => {
+    const pattern = ['foobar', /foo/];
```

Comment:
You should probably change this to something that's not overlapping.  Like 'foo' and /bar/

Reply:
We fixed this.